lib/services/event_service.ex

```diff
@@ -51,6 +51,20 @@ defmodule Aprb.Service.EventService do
                           ]
                         }]",
           unfurl_links: true }
+      "bidding" ->
+        %{
+          text: ":gavel: #{event["type"]} on #{event["lotId"]} from Paddle #{event["bidder"]["paddleNumber"]} ",
+          attachments: "[{
+                          \"fields\": [
+                            {
+                              \"title\": \"Amount\",
+                              \"value\": \"#{format_price(event["amountCents"] || 0)}\",
```

Comment:
We don't want dollar signs here. Could you add a method that doesn't have currency? I don't want to confuse people or make them thing it's wrong.